tests/integration/test_set_environment_variable.py

```diff
@@ -81,14 +80,14 @@ def run_test(self, mocked_runner, tmpdir, cli_entry, config_fixture, expected):
 
         params = shlex.split(cli_entry) + ["--pp", "never"]
 
-        with mock.patch("sys.argv", params):
-            with mock.patch.dict(os.environ, {"ANSIBLE_NAVIGATOR_CONFIG": cfg_path}):
-                with mock.patch.dict(
-                    os.environ,
-                    {"ANSIBLE_NAVIGATOR_COLLECTION_DOC_CACHE_PATH": coll_cache_path},
-                ):
-                    with pytest.raises(Exception, match="called"):
-                        cli.main()
+        mocker.patch("sys.argv", params)
+        mocker.patch.dict(os.environ, {"ANSIBLE_NAVIGATOR_CONFIG": cfg_path})
+        mocker.patch.dict(
```

Comment:
I'd argue that this should've been done with `monkeypatch` rather than `mocker`.

Reply:
agree! thx

Reply:
The change to monkeypatch is complete